standards_lab/api/views.py

```diff
@@ -158,6 +172,19 @@ def delete(self, *args, **kwargs):
         return OK(project)
 
 
+class OwnedProjectConfig(View):
+    """DELETE will delete the whole project
+    (The DELETE method in ProjectConfig is already used for something else so it can't be there.)
+    """
```

Comment:
I'd prefer if we did a refactor of  `Project*File` in `api:views` first and _then_ add new code rather than add more code to be refactored at a later date (the old "Don't put off until tomorrow what you can do today" applies I think). We have the time at the moment to fix/do ground work and then build upon it. If you don't feel familiar enough with what needs refactoring though I'm happy to do that or I can elaborate on it more.

Reply:
I'm happy to do that refactoring